### ISSUE TITLE
Installation of v.0.40.0 fails with `go install`

### ISSUE THREAD

**Issue by cdillond** [jesseduffield/lazygit#2885]
**Describe the bug**
Installation of lazygit v0.40.0 using `go install` fails due to a `replace` directive in the go.mod file.

**To Reproduce**
1. Run `go install github.com/jesseduffield/lazygit@latest`
2. When I do this, the following message appears: 
```
go: github.com/jesseduffield/lazygit@latest (in github.com/jesseduffield/lazygit@v0.40.0):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

**Expected behavior**
I expected lazygit to install successfully.

**Version info:**
I was able to avoid the issue by running `go install github.com/jesseduffield/lazygit@v0.39.4` instead.

**Additional context**
Per the [Go docs](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies) (and [this other github issue](https://github.com/golang/go/issues/44840#issuecomment-1651863470)), `go install` doesn't work with `replace` directives.

I think this might be the source of the problem. 
[`replace github.com/gdamore/tcell/v2 v2.6.0 => github.com/stefanhaller/tcell/v2 v2.6.16`](https://github.com/jesseduffield/lazygit/blob/7847203d6e8e89d4f5dc52ac167a219b0bd69d5f/go.mod#L76)

**Comment by stefanhaller** [comment]
This is being discussed in https://github.com/golang/go/issues/44840, it's a problem in golang's module system that people seem to disagree about whether or how it should be solved.

For the time being, the solution seems to be to not use a replace directive for tcell, but import the fork for real. Starting work on this in https://github.com/jesseduffield/gocui/pull/37.

**Comment by stefanhaller** [comment]
While this should be solved by #2888, it will of course still not work until we make a new release.
